src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -1410,27 +1410,10 @@ private string ReadLineFromFile(string initialContent)
             return sb.ToString();
         }
 
+#if UNUX
```

Comment:
```suggestion
#if UNIX
```
That might be a bit tricky 😉